src/lib/operators/aggregate.cpp

```diff
@@ -107,13 +107,9 @@ struct GroupByContext : ColumnVisitableContext {
 /*
 Compares two AggregateKeys, but considers two NULL values to be equal. This is so that they form a single group.
 */
-struct NullEqualTo {
+struct EqualTo {
```

Comment:
There is no need for an explicit NULL compare because NULLs are represented as uint64_t{0} in the AggregateKey now. So it still returns true if both values are NULL.

Reply:
And this `uint64_t{0}` is guaranteed to not conflict with actual values?

Reply:
Yes, I'm generating IDs for equal values in a column. And `uint64_t{0}` is reserved for NULL.